### DISCUSSION TITLE
Package Logos

### DISCUSSION THREAD

**Discussion by danvick** [danvick/flutter_form_builder#1285] [Polls]
Hi everyone,
I've been thinking of designing a logo for the packages under this ecosystem for a while now. 

I've come up with these 2:

| ![Blue Blocks Internet Logo](https://github.com/flutter-form-builder-ecosystem/flutter_form_builder/assets/7383575/852389c5-e839-4bc5-a61f-c2f4ab9759a7) |![Blue Blocks Internet Logo(1)](https://github.com/flutter-form-builder-ecosystem/flutter_form_builder/assets/7383575/5c19ad33-4018-4cca-95d8-0063f07f045e) |
|----------|-----------|
| Logo 1    | Logo 2      |

The point is to have one logo for all packages to take the same branding but have the tagline differentiate them

**Comment by deandreamatias** [comment]
I think that a logo with Flutter icons would be nice, but I don't know if any of these two initial options, can adapt easy to a square/circle icon

**Reply by danvick** [reply]
The plan is to have something like a favicon to be just the top square part only:

![SCR-20230824-l60](https://github.com/flutter-form-builder-ecosystem/flutter_form_builder/assets/7383575/c66d9c74-dede-4024-b8dc-9ec6d824367c)
